style.css

```diff
@@ -0,0 +1,55 @@
+* {
+  background-color: gainsboro;
+}
+div {
+  text-align: center;
+}
```

Comment:
If you remove the `div`s as I suggest, you can instead apply the `text-align` rule to your `body` element (since all the text on your page is centered) 👍🏻